### PR TITLE
Improving perm space p12

### PR DESCRIPTION
**Pull request by tesonep**

### CODE REVIEW

File: smalltalksrc/VMMaker/AbstractComposedImageAccess.class.st [hunk 1]
```diff
@@ -20,7 +20,7 @@ AbstractComposedImageAccess >> createImageDirectory: imageFileName [
 			self
 				cppIf: ((self defined: #_WIN32) or: [(self defined: #_WIN64)])
 				ifTrue: [ self mkdir: imageFileName ]
-				ifFalse: [ self mkdir: imageFileName _: 0777 ] ]
+				ifFalse: [ self mkdir: imageFileName _: 8r777 ] ]
```
Comment (guillep) [inline smalltalksrc/VMMaker/AbstractComposedImageAccess.class.st:23]: :^) @PalumboN

File: smalltalksrc/VMMaker/Spur32BitMMLESimulator.class.st [hunk 2]
```diff
@@ -83,7 +83,7 @@ Spur32BitMMLESimulator >> fetchFloatAt: floatBitsAddress into: aFloat [
 	aFloat at: 1 put: (self uint32AtPointer: floatBitsAddress+4)
 ]
 
-{ #category : #'object access' }
+{ #category : #'as yet unclassified' }
```
Comment (guillep) [inline smalltalksrc/VMMaker/Spur32BitMMLESimulator.class.st]:
```suggestion
{ #category : #'object access' }
```

File: smalltalksrc/VMMaker/Spur64BitMMLESimulator.class.st [hunk 3]
```diff
@@ -83,7 +83,7 @@ Spur64BitMMLESimulator >> fetchFloatAt: floatBitsAddress into: aFloat [
 	aFloat at: 1 put: (self uint32AtPointer: floatBitsAddress+4)
 ]
 
-{ #category : #'object access' }
+{ #category : #'as yet unclassified' }
```
Comment (guillep) [inline smalltalksrc/VMMaker/Spur64BitMMLESimulator.class.st]:
```suggestion
{ #category : #'object access' }
```

File: smalltalksrc/VMMaker/Spur32BitMMLESimulator.class.st (hunk 2 above)
Comment (PalumboN) [reply smalltalksrc/VMMaker/Spur32BitMMLESimulator.class.st]: This is a bug from the automatic protocol selection I think. We can improve it in the next sprint.

File: smalltalksrc/VMMaker/AbstractComposedImageAccess.class.st (hunk 1 above)
Comment (PalumboN) [reply smalltalksrc/VMMaker/AbstractComposedImageAccess.class.st:23]: ~~1100001001~~ `111111111` 👍

**Review by PalumboN — APPROVED** [review]

**Pull request merged**